MLS.Agent.Tools/CommandLineResultExtensions.cs

```diff
@@ -12,7 +12,7 @@ public static void ThrowOnFailure(this CommandLineResult result, string message
         {
             if (result.ExitCode != 0)
             {
-                throw new CommandLineInvocationException(result, message ?? string.Join("\n", result.Error.Concat(result.Output)));
+                throw new CommandLineInvocationException(result, message + "\n" + string.Join("\n", result.Error.Concat(result.Output)));
```

Comment:
interpolated string, also environment.newline

Reply:
also previously message could be null?

Reply:
Done. Thanks!